compiler/AST/ModuleSymbol.cpp

```diff
@@ -651,6 +651,18 @@ void ModuleSymbol::addDefaultUses() {
   }
 }
 
+// Helper function for computing the index in the module use list
+// within mod for a use of usedModule
+static int moduleUseIndex(ModuleSymbol* mod, ModuleSymbol* usedModule) {
```

Comment:
What's the argument for this not being a private method on ModuleSymbols?

Reply:
That's one of those style things - why not make it a static function just used in this file? :)

I tend to prefer this strategy to private methods because then I don't have to muck about in header files (which I feel should give the public API) when changing it.